docker-images/kafka/cruise-control-scripts/cruise_control_config_generator.sh

```diff
@@ -43,12 +43,16 @@ cluster.configs.file=$CC_CLUSTER_CONFIG_FILE
 webserver.accesslog.path=$CC_ACCESS_LOG
 webserver.http.address=0.0.0.0
 webserver.http.cors.allowmethods=OPTIONS,GET
+webserver.ssl.keystore.location=/tmp/cruise-control/client-server.keystore.p12
```

Comment:
How does this deal with the TLS hostname verification when you use the same secret for both connecting to Kafka and as a server certificate for the webserver?

Reply:
Thanks for pointing this out. I have now added Cruise Control's public certificate to the truststore of the KafkaRebalance client so that TLS hostname verification is preformed

Reply:
Maybe some better name than `client-server` would be better? Is it a client certificate? Server certificate? What client and what server? Maybe it should be called `cruise-control.keystore.p12`?